kqueen/config/auth.py

```diff
@@ -0,0 +1,25 @@
+from kqueen.config import current_config
+
+
+class AuthModules():
```

Comment:
Is there a reason to create this class?

Reply:
to increase visibility and avoid case of multiple value-entries in config file, when 
a = 'One'
b = a + 'Two' == 'One' + 'Two'

apply_changes:
a == 'EnvOne'
b == 'One' + 'Two'

Reply:
we already discussed the case with u previously

Reply:
and about class, i wanted to provide simple possibility to add another auth modules, if u have any other good solutions, please go ahead